example/tests/test_models.py

```diff
@@ -1,13 +1,26 @@
 from django.contrib.auth.models import User
 from django.test import TestCase, override_settings
-from django.utils.translation import ugettext_noop, activate
+from django.utils.translation import override, ugettext_noop
 
 from pynotify.exceptions import MissingContextVariableError
-from pynotify.models import Notification, NotificationTemplate
+from pynotify.models import AdminTemplate, Notification, NotificationTemplate
 
 from articles.models import Article
 
 
+class AdminTemplateTestCase(TestCase):
+
+    def test_admin_template_should_have_string_representation(self):
+        template = AdminTemplate.objects.create(
+            title='Hello!',
+            slug='test-template',
+        )
+        self.assertEqual(str(template), 'admin notification template #{} ({})'.format(
+            template.pk,
+            template.slug,
+        ))
```

Comment:
this test seems pointless, but I guess 100% coverage is 100% coverage

Reply:
:D:D

Reply:
Well `__str__` is also a legit method with its logic, it should be tested :-)